### PR TITLE
feat(pdf): Move annotation styling to box-annotations

### PR DESCRIPTION
**Pull request by karelee7**
**Description of issue:**
In v3.6.172 of PDFjs, a z-index styling was explicitly added to `pdf_viewer.css`: https://cdnjs.cloudflare.com/ajax/libs/pdf.js/3.6.172/pdf_viewer.css. This styling was not present in the previous version of PDFjs that we're using: https://cdnjs.cloudflare.com/ajax/libs/pdf.js/2.14.305/pdf_viewer.css and keeping this new styling causes issues with the other layers in preview (i.e. popup layer, highlighted layer, etc.). Updating styling for our layers resolves the issues that QA has discovered, but moving them to the box-annotations repo is preferable. 

Have tested this change in the SDK with the stylings moved to box-annotations/linked with EUA.

### CODE REVIEW

**Comment by jstoffan** [comment]
@karelee7, it looks like we're in a familiar game of whack-a-mole with z-index. I have a couple concerns:

- We didn't have this problem before the pdf.js upgrade. We relied mostly on the browser's default stacking context. What changed?
- Should this styling live in `box-annotations`, instead?

**Comment by karelee7** [comment]
@jstoffan Yeah, I know that in the new PDFjs upgrade, one thing that changed was that the `textLayer`'s z-index changed. I think that was the only update to z-index, and I think caused some issues with the other layers that we have. Hm, yeah, I'll take a look at box-annotations and see if it fits better over there

**Comment by jstoffan** [comment]
@karelee7, why did the z-index of the text layer change? Was it to support PDF annotations? Can we undo it safely?

**Comment by bfoxx1906** [comment]
@karelee7 I was just about to say the same thing Jared did about the constant Z-index issues. We keep finding new issues. Have we looked through the code at all the z-index values  set in the css and verified that they are still valid?

**Comment by karelee7** [comment]
@bfoxx1906 Yeah, it's a good point, looking through it now

File: src/third-party/doc/2.93.0/pdf_viewer.min.css [hunk 1]
```diff
@@ -1 +1 @@
...
...izontal,.scrollWrapped) .spread{margin-inline:0}.pdfViewer:is(.scrollHorizontal,.scrollWrapped) :is(.page,.spread),.spread :is(.page,.dummyPage){display:inline-block;vertical-align:middle}.pdfViewer:is(.scrollHorizontal,.scrollWrapped) .page,.spread .page{margin-inline:var(--spreadHorizontalWrapped-margin-LR)}.pdfViewer.removePageBorders .spread .page,.pdfViewer.removePageBorders:is(.scrollHorizontal,.scrollWrapped) .page{margin-inline:5px}.pdfViewer .page canvas{margin:0;display:block}.pdfViewer .page canvas .structTree{contain:strict}.pdfViewer .page canvas[hidden]{display:none}.pdfViewer .page canvas[zooming]{width:100%;height:100%}.pdfViewer .page.loadingIcon::after{position:absolute;top:0;left:0;content:"";width:100%;height:100%;background:url(images/loading-icon.gif) center no-repeat;display:none;transition-property:display;transition-delay:var(--loading-icon-delay);z-index:5;contain:strict}.pdfViewer .page.loading::after{display:block}.pdfViewer .page:not(.loading)::after{transition-property:none;display:none}.pdfPresentationMode .pdfViewer{padding-bottom:0}.pdfPresentationMode .spread{margin:0}.pdfPresentationMode .pdfViewer .page{margin:0 auto;border:2px solid transparent}
```
Comment (bfoxx1906) [inline src/third-party/doc/2.93.0/pdf_viewer.min.css:1]: did we mean to change this file?

Comment (karelee7) [reply src/third-party/doc/2.93.0/pdf_viewer.min.css:1]: No, I tried copy/pasting the original minified file but it didn't show up as a change in VS Code, lemme try restarting VS Code/re-copying the original minified file again..

Comment (karelee7) [reply src/third-party/doc/2.93.0/pdf_viewer.min.css:1]: Hm, no luck reverting, I'm going to close this PR and open a new one to see if that fixes this diff issue!

**Pull request closed without merging**